int32 roboyID
int32 leg
string joint
geometry_msgs/Point force
geometry_msgs/Point torque